ner_v2/detectors/numeral/number_range/standard_number_range_detector.py

```diff
@@ -282,7 +282,7 @@ def _detect_min_num_range_with_prefix_variants(self, number_range_list=None, ori
 
         if self.min_range_prefix_variants:
             min_prefix_choices = '|'.join(self.min_range_prefix_variants)
-            min_range_start_pattern = re.compile(ur'((?:{min_prefix_choices})\s+({number}\d+))'.format(
+            min_range_start_pattern = re.compile(u'((?:{min_prefix_choices})\\s+({number}\\d+))'.format(
```

Comment:
Use re.escape.
```
min_range_start_pattern = re.compile(u'((?:{re.escape(min_prefix_choices)})\\s+({re.escape(number)}\\d+))'.format(number=numeral_constant.NUMBER_REPLACE_TEXT, min_prefix_choices=min_prefix_choices), re.UNICODE)
```

Reply:
The pattern is already using re.escape in `_init_regex_for_range` method defined in standard_number_range_detector.py: 

```
self.min_range_prefix_variants = [re.escape(variant) for variant, value in self.range_variants_map.items()
                                          if (value.position == -1 and
                                              value.range_type == numeral_constant.NUMBER_RANGE_MIN_TYPE)]

```